spec.md

```diff
@@ -725,7 +725,11 @@ Plugins MAY create 3 types of volumes:
 
 - Empty volumes. When plugin supports `CREATE_DELETE_VOLUME` OPTIONAL capability.
 - From an existing snapshot. When plugin supports `CREATE_DELETE_VOLUME` and `CREATE_DELETE_SNAPSHOT` OPTIONAL capabilities.
+  SP SHOULD create a volume that looks like exact copy of the original snapshotted volume at the time the snapshot was taken.
```

Comment:
> No, CO requested 2 GiB and SP MUST provide at least 2GiB. But it does not say anything about size of the actual filesystem on the volume. It may be (and IMO should) still be 1GiB. For filesystem volumes, containers will see only this 1 GiB filesystem.

Isnt this conflict with the example above? We should clarify the content should be the exact copy but SP can return with a larger size of Volume, although only a portion of it might be used